### DISCUSSION TITLE
Merging a Branch

### DISCUSSION THREAD

**Discussion by iLakshya** [community/community#39808] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Which option is better to merge a branch through the terminal or through the GitHub website?

**Comment by radar-blip** [comment]
I think it's more a matter of personal preference.

**Reply by airtower-luna** [reply]
One thing to add: If you merge locally (on the terminal) and then push the merged branch, a PR for that merge should be marked as merged automatically. So you can have both, in a way.

**Reply by radar-blip** [reply]
I object.

**Reply by airtower-luna** [reply]
Object to what? In my experience that process (merge locally, push, PR marked as merged) works, but admittedly it's been a while since I've used it. :sweat_smile:

**Reply by radar-blip** [reply]
I object to you having both, it's either one or the other. 😄

**Reply by airtower-luna** [reply]
Okay, that's true, it *is* a local merge, just reflected in a PR you might have. :smile_cat:
